packages/frontend/scss/message/_message.scss

```
@@ -211,6 +211,7 @@ $info-text-max-width: 400px;
   .metadata {
     margin-top: 10px;
     margin-bottom: -7px;
+    pointer-events: all;
```

`all` seems to be SVG-only.

```suggestion
    pointer-events: initial;
```